src/components/Mastodon.vue

```diff
@@ -46,167 +46,46 @@
-      <header class="card-header">
-        <h2 class="card-title">Masto Test</h2>
-      </header>
-      <section class="card-body">
-        <article class="card-tile" v-for="number in articleSet" :key="number">
-          <a class="article-anchor-wrapper" :href="articleArray[number]?.article_url">
-            <span class="article-image-wrapper" :style="{ 'background-image': 'linear-gradient(180deg, rgba(0,0,0,0) 0%, rgba(46,46,46,0.5480786064425771) 35%, rgba(0,0,0,1) 100%), url(' + articleArray[number]?.article_logo + ')' }">
-              <img class="article-logo" :src="articleArray[number]?.article_logo" alt="">
-              <h5 class="article-title"><pre>{{ articleArray[number]?.article_body }}</pre></h5>
-              <img class="article-image" :src="articleArray[number]?.article_image" alt="">
-              <p class="p3 article-publisher">{{ articleArray[number]?.article_publisher }} || {{ articleArray[number]?.article_published_date }}</p>
-            </span>
-          </a>
-        </article>
-      </section>
-    </article>
-</template>
-
-<style scoped lang="scss">
-  .card_mastodon-feed {
-  
-//    flex: 1 0 300px;
-//    width: 100%;
-//    min-height: 27rem;
-//    max-height: 100rem;
-//    overflow: scroll;
-    
-    p {
-      color: $background-lighter;
-      text-align: center;
-    }
-    
-    .p3 {
-      font-size: 0.8rem;
-    }
-    
-    pre {
-      white-space: pre-wrap;
-    }
-    
-    .blurred-lines {
-      display: block;
-//      width: 100%;
-//      height: 100%;
-      backdrop-filter: blur(10px);
-//      padding-top: $card-padding-internal;
-
-
-    }
-    
-    .article-wrapper {
-      align-items: flex-start;
-      display: flex;
-      flex-wrap: wrap;
-      justify-content: space-between;
-      margin-top: $card-padding-internal;
-    }
-    
-    .article-anchor-wrapper {
-      display: block;
-      position: relative;
-//      max-width: 100%;
-//      min-width: 100%;
-//      width: 100%;
-//      height: 100px;
-
-      &::after {
-        content: '';
-        display: block;
-        width: 100%;
-        height: 100%;
-        position: absolute;
-        top: 0;
-        left: 0;
-        z-index: 0;
-      }
-
-
-}
-    
-    .card-header {
-      background-color: #2F0C7A;
-      color: #858AFA;
-    }
-    
-    .article-title {
-      margin: 0.4rem $card-padding-internal $card-padding-internal $card-padding-internal;
-      color: $background-lighter;
-      text-shadow: 0 0 12px #000000, 0 0 2px $background;
-      text-align: center;
-      text-decoration: none;
-    }
-    
-    .article-logo {
-
-      display: block;
-//      max-width: 60px;
-      height: 60px;
-      aspect-ratio: 1/1;
-      object-fit: cover;
-
 
-      margin: 0 auto calc($card-padding-internal * 3) auto;
-      filter: drop-shadow(0 0 12px #000000) drop-shadow(0 0 2px $background);
+  <article class="card-tile" v-for="number in articleSet" :key="number">
```

Comment:
didnt look deeply into this but im wondering if some modification here (or a similar place in a diff component) is where our errors are coming from

Reply:
💯

Reply:
I'm... unsure. We'll keep an eye on it elsewhere.